workers/notifier/types/channel.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Possible channel types
+ */
+export enum ChannelType {
+  Email = 'email',
+  Telegram = 'telegram',
+  Slack = 'slack',
+}
+
+/**
+ * Notification channel object
+ */
+export interface Channel {
+  /**
+   * Shows if channel is enabled
+   */
+  readonly isEnabled: boolean;
+
+  /**
+   * Channel endpoint
+   */
+  readonly endpoint?: string;
```

Comment:
по API, endpoint не может отсутствовать 
```suggestion
  readonly endpoint: string;
```

Reply:
Так а если он не указан, он чему равен?

Reply:
в смысле не указан?
Пользователь при создании правила не может оставить endpoint пустым

Reply:
То есть channel может быть не указан?

Reply:
да
какие-то из каналов могут быть пустыми, но поля внутри канала всегда должны быть

Reply:
Ок